src/header.tsx

```diff
@@ -135,6 +135,8 @@ type HeaderLayoutProps = {
     sideBySideExtraOnDesktop?: boolean;
     children?: void;
     dataAttributes?: DataAttributes;
+    bleed?: boolean;
+    bleedValue?: number;
 };
 
```

Comment:
I found a bug when you use a card inside extra prop + bleed prop
![image](https://user-images.githubusercontent.com/6722153/223448073-1ff740cd-5b7e-4c33-b4c2-8b0ea65c565b.png)